data/json/dreams.json

```diff
@@ -276,7 +286,12 @@
   },
   {
     "type": "dream",
-    "messages": [ "You dream of a home in the forests.", "You feel beautiful, and yet riven with worry..." ],
+    "messages": [
+      "You dream of a home in the forests.",
+      "You dream of drinking flowered wine.",
```

Comment:
This is crossing the line into fantasy elf.
```suggestion
```

Reply:
Yeah I think there is some canon I'm missing here - like what type of fairie folk are not fantasy?  These Elf-A aren't really elves, they are plant bird people?